docs/setup/independent/troubleshooting-kubeadm.md

```diff
@@ -34,14 +34,32 @@ If you see the following warnings while running `kubeadm init`
 
 Then you may be missing ebtables and ethtool on your Linux machine. You can install them with the following commands: 
 
+- For ubuntu/debian users, try `apt install ebtables ethtool`.
+- For CentOS/Fedora users, try `yum install ebtables ethtool`.
+
+#### kubeadm blocks waiting for `control plane` during installation
+
+If you notice that `kubeadm init` hangs after printing out the following line
+
 ```
-# For ubuntu/debian users, try 
-apt install ebtables ethtool
+[apiclient] Created API client, waiting for the control plane to become ready                          
+```
+
+You may want to first check if your node has network connection problem.
```

Comment:
I would change this to:

```
This may be caused by a number of problems. The most common are:

- network connection problems. Check that your machine has full network connectivity before continuing.
- the default cgroup driver configuration for the kubelet differs from that used by Docker
- control plane Docker containers are crashlooping or hanging. You can check this by running `docker ps` and investigating each container by running `docker logs`.

... other stuff about /var/log/message
```